src/chaos.js

```diff
@@ -52,4 +91,64 @@ export class KubernetesClient {
         this.kubernetes.namespaces.delete(randomNamespace.name)
         return randomNamespace.name
     }
-}
\ No newline at end of file
+}
+
+export class StressNodesAttack {
+    constructor(client) {
+        this.kubernetes = client
+        this.name = 'stress-nodes'
+        this.namespace = 'default'
+        this.cpu = '0'
+        this.io = '0'
+        this.vm = '0'
+        this.vmBytes = '256MB'
+        this.timeout = '10'
+    }
+
+    inNamespace(namespace) {
+        this.namespace = namespace
+        return this
+    }
+    inNodes(nodes) {
+        this.nodes = nodes
+        return this
+    }
+    withName(name) {
+        this.name = name
+        return this
+    }
+    withCPU(cpu) {
+        this.cpu = cpu
+        return this
+    }
+    withIO(io) {
+        this.io = io
+        return this
+    }
+    withVM(vm) {
+        this.vm = vm
+        return this
+    }
+    withVMBytes(vmBytes) {
+        this.vmBytes = vmBytes
+        return this
+    }
+    withTimeout(timeout) {
+        this.timeout = timeout
+        return this
+    }
+    start() {
+        this.kubernetes.runJobOnNodes(
+            this.namespace,
+            this.name,
+            this.nodes,
+            'progrium/stress',
```

Comment:
From what I've seen, this tool is no longer maintained. We maybe should migrate to use [`stress-ng`](https://wiki.ubuntu.com/Kernel/Reference/stress-ng). We can do this in a follow up MR.